cmd/dalc/main.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func main() {
+	root := rootCmd()
+	if err := root.Execute(); err != nil {
+		os.Exit(1)
+	}
+}
+
+func rootCmd() *cobra.Command {
+	rootCmd := &cobra.Command{
+		Short: "data availability light client for celestia",
+		PersistentPreRunE: func(cmd *cobra.Command, _ []string) error {
+			return nil
+		},
+	}
+	return rootCmd
+}
```

Comment:
tbh just put this here to have some go code in the repo, otherwise none of the CI tests work